word-count/defaults.yaml

```diff
@@ -0,0 +1,33 @@
+helm-app:
+  namespace: ${NAMESPACE}
+
+streams-bootstrap:
+  version: "3.1.0"
+  values:
+    labels:
+      pipeline: ${pipeline.name}
+    kafka:
+      bootstrapServers: ${config.kafka_brokers}
+      schemaRegistryUrl: ${config.schema_registry.url}
+      optimizeLeaveGroupBehavior: false
```

Comment:
Why bother configuring it?

Reply:
We don't need SR 😄 The app only has (String, String) records.

Reply:
But then remove optimizeLeaveGroupBehavior

Reply:
Done